### ISSUE TITLE
[20230326] Weekly AI ArXiv 만담 시즌2 - 11회차

### ISSUE THREAD

**Issue by scene-the-ella** [jungwoo-ha/WeeklyArxivTalk#77]

**Comment by jungwoo-ha** [comment]
### News
- Conferences
  - ICML 2023 and ACL 2023 Rebuttal 모두 수고 많으셨습니다.
  - ICML author-reviewer discussion 기간이에요~ (26일까지)
- [ChatGPT Plugin]
  - ChatGPT 생태계가 만들어지는 큰그림 (Text-to-any app action)
  - ChatGPT가 API를 호출할 수 있도록 만들어 그 자체로 다양한 앱을 연동토록 가능
  - 다양한 앱들을 통해 개인의 (앱활용) 데이터는 물론 ChatGPT를 통해 NLP-to-모든 액션이 가능하게...
  -  Third party Plugin 개발자들에게 새로운 application 기회를 제공, 이제 LM자체가 OS화
  - 그렇게 해서 쌓인 NL-to-action 데이터들은 다시 ChatGPT 고도화에 활용...
  - 브라우저(WebGPT를 한참 넘어선)와 python interpreter 제공
  - [Retrieval plugin](https://github.com/openai/chatgpt-retrieval-plugin)은 오픈소스로 제공
  - Partner third-party
![image](https://user-images.githubusercontent.com/11782739/227569038-d91732e3-005c-479c-a14e-365992ca898c.png)


### ArXiv

**Comment by veritas9872** [comment]
**_뉴스_**:
**ChatGPT Plugins**
Blog: https://openai.com/blog/chatgpt-plugins

ChatGPT에서 인터넷 접속 및 외부 앱 연결을 위한 plugin API를 공개했습니다.
현재 안전 이슈 등으로 소수의 유저에게만 공개하고 있으며 waiting list에 등록해야 합니다.

**Scaling in the service of reasoning & model-based ML**
Blog: https://yoshuabengio.org/2023/03/21/scaling-in-the-service-of-reasoning-model-based-ml

Yoshua Bengio 옹께서 Large Language Model이 System 2 cognition을 하기 위해 현재 LLM의 한계 및 앞으로 나아갈 방향에 대해 논의합니다.

**Hello Dolly: Democratizing the magic of ChatGPT with open models**
Blog: https://www.databricks.com/blog/2023/03/24/hello-dolly-democratizing-magic-chatgpt-open-models.html

DataBricks 연구팀에서 GPT-J 6B 모델을 기반으로 작은 fine-tuning dataset에 Instruction find-tuning을 실행했을 때 무려 30분만에(!)  ChatGPT와 유사하게 instruction에 대해 응답하도록 유도할 수 있었습니다. GPT-J 6B는 공개되고 2년이 지난 모델이라는 것을 감안한다면 Instruction에 대해 답변하는 것은 거대 모델의 역량도 중요하지만 fine-tuning 과정 또한 매우 중요하다는 것을 보여줍니다.


**_Software_**:
**RAFT: Reusable Accelerated Functions and Tools**

GitHub: https://github.com/rapidsai/raft
Docs: https://docs.rapids.ai/api/raft/stable
Blog: https://developer.nvidia.com/blog/reusable-computational-patterns-for-machine-learning-and-data-analytics-with-rapids-raft


**_Research_**:

**When and Why Vision-Language Models Behave like Bags-Of-Words, and What to Do About It?**
OpenReview: https://openreview.net/forum?id=KRLUvxh8uaX
GitHub: https://github.com/mertyg/vision-language-models-are-bows
Blog: https://towardsdatascience.com/your-vision-language-model-might-be-a-bag-of-words-30b1beaef7f8

![image](https://user-images.githubusercontent.com/33523965/227712593-42854822-8ace-4a18-bb36-65f2836a3e09.png)

조~금 오래되었지만 ICLR2023 Oral Session 선정 논문이어서 공유합니다.
해당 연구에서는 기존CLIP과 같은 Vision Language Model (VLM)은 단어의 순서에 영향을 받지 않는 Bag-of-Words 모델과 유사한 행동을 보이며 이런 문제를 해결하기 위해 어순의  영향을 받는 hard negative case mining을 적용할 경우 문장의 의미를 이해하도록 학습할 수 있음을 보여줍니다.

**Erasing Concepts from Diffusion Models**
ArXiv: https://arxiv.org/abs/2303.07345
GitHub: https://github.com/rohitgandikota/erasing

![image](https://user-images.githubusercontent.com/33523965/227712746-5fd2866f-17c2-430a-8a0f-e2c254669e58.png)

MIT David Bau 교수님 연구실에서 나온 최근 연구입니다. 
최근 Text-to-Image diffusion 모델에서 바람직하지 않은 출력물을 생성할 수 있는 문제가 대두되고 있는데 모델 weight 자체에서 원치 않는 개념을 제거할 방법을 제시합니다.

![image](https://user-images.githubusercontent.com/33523965/227712834-c5b78d0d-c781-4bbd-8fb1-073d6107bf1c.png)

하나의 Frozen Stable Diffusion (SD) 모델과 하나의 Erased Stable Diffusion (ESD) 모델을 활용하여 원치 않는 개념에 대한 prompt를 제공한 경우와 제공하지 않은 경우에 대해 일치하도록 학습함으로써 추후 저작권 등을 위배하는 prompt를 제공하더라도 마치 해당 개념이 모델에서 삭제된것과 마찬가지의 출력을 제공하도록 합니다.

**Comment by gyunggyung** [comment]
github.com/gyunggyung

**Comment by jwlee-neubla** [comment]
### Arxiv

#### [The effectiveness of MAE pre-pretraining for billion-scale pretraining](https://arxiv.org/abs/2303.13496)
- Meta AI
- MAE를 이용하여 computer vision에서 늘 사용되는 pretrain-then-finetune paradigm을 개선
- pre-pretraining : MAE를 이용한 SSL(1 epoch) -> weak and noisy label을 이용한 WSP(Weakly Suprevised Pretraining)
- MAE가 model 뿐만 아니라 data에도 scalabilty를 가짐
- iNaturalist-18(91.3%), 1-shot ImageNet-1k(62.1%, zero-shot transfer on Food-101(96.0%)에서 SOTA 달성

<img width="695" alt="image" src="https://user-images.githubusercontent.com/93505462/227773740-7cb9ea96-ce4d-47ea-aec4-6fb11594a590.png">
<img width="1409" alt="image" src="https://user-images.githubusercontent.com/93505462/227773795-12ed448b-725a-4efa-9e4b-7f9867f56985.png">
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/93505462/227773844-3c5afcc5-463b-441b-9a70-2cf77843a575.png">
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/93505462/227773861-e19fb682-8dca-4973-aa1d-0164e142d588.png">
<img width="802" alt="image" src="https://user-images.githubusercontent.com/93505462/227773887-e0419ac4-26ca-4bc5-b2b3-8c0586345a3b.png">


#### [SmoothQuant: Accurate and Efficient Post-Training Quantization for Large Language Models](https://arxiv.org/abs/2211.10438)
- MIT + Nvidia
- LLM quantization에서 activiation에 outlier가 존재해서 어려움
- LLM.int8()은 outlier를 FP16으로 처리했는데, memory는 줄일 수 있지만 속도가 느림
- activation outlier로 인한 quantization difficulty 를 weight 로 좀 넘겨주자 
<img width="515" alt="image" src="https://user-images.githubusercontent.com/93505462/227774160-d5bf9b7a-2f1d-42cc-a033-87f3bf5b2a23.png">
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/93505462/227774196-a56be03f-1a2f-431b-a3a8-860ea9f2050e.png">
<img width="528" alt="image" src="https://user-images.githubusercontent.com/93505462/227774216-275e8804-7d0f-48a5-9a9b-173be330dcdf.png">
<img width="504" alt="image" src="https://user-images.githubusercontent.com/93505462/227774347-cbf804df-cd73-4458-b4e5-af567693bf61.png">
<img width="505" alt="image" src="https://user-images.githubusercontent.com/93505462/227774228-037f49d0-21e1-4ab1-916a-e726885b09e2.png">
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/93505462/227774410-6821235b-4aca-46e1-bd5c-a05d9e5a606b.png">

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/93505462/227774253-78297c0a-afcf-4ca1-bb6e-85caaa548b00.png">
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/93505462/227774271-d7b4b07e-39f9-4874-b300-07af00f37d81.png">

**Comment by nick-jhlee** [comment]
For my small comment:

[Efficient fair PCA for fair representation learning](https://arxiv.org/pdf/2302.13319.pdf) (AISTATS 2023)
- from Amazon AWS
- efficient ver of fair PCA
- (see Fig 5)

**Comment by nick-jhlee** [comment]
For next time:

[Sparks of Artificial General Intelligence: Early experiments with GPT-4](https://arxiv.org/abs/2303.12712) (Microsoft Research)

교훈: latex에서 comment를 뺍시다!

“Given the breadth and depth of GPT-4's capabilities, we believe that it could reasonably be viewed as an early (yet still incomplete) version of an artificial general intelligence (AGI) system.”



[Why Can GPT Learn In-Context? Language Models Secretly Perform Gradient Descent as Meta-Optimizers](https://arxiv.org/abs/2212.10559) (arXiv 2022)


[The Learnability of In-Context Learning](https://arxiv.org/abs/2303.07895)
